src/monitor/pgautofailover.sql

```diff
@@ -579,3 +600,63 @@ comment on function pgautofailover.set_node_replication_quorum(int, text, int, b
 grant execute on function
       pgautofailover.set_node_replication_quorum(int, text, int, bool)
    to autoctl_node;
+
+
+create function pgautofailover.synchronous_standby_names
```

Comment:
can you reflect changes to upgrade script ?

Reply:
Done now, thanks.